plugins/discovery-gce/build.gradle

```diff
@@ -52,6 +52,12 @@ tasks.named("thirdPartyAudit").configure {
           'org.apache.avalon.framework.logger.Logger',
           'org.apache.log.Hierarchy',
           'org.apache.log.Logger',
+          'org.apache.avalon.framework.logger.Logger',
```

Comment:
I guess we only need jms.Message? the rest are already declared?

Reply:
Let me check that

Reply:
You're right! We should add only one `javax.jms.Message` entry here